engine/class_modules/sc_death_knight.cpp

```diff
@@ -3957,14 +3961,8 @@ struct helchains_buff_t : public buff_t
     } );
   }
 
-  // Helchains ticks twice on buff application and buff expiration, hence the following overrides
-  // https://github.com/SimCMinMax/WoW-BugTracker/issues/390
-  void execute( int stacks, double value, timespan_t duration ) override
-  {
-    buff_t::execute( stacks, value, duration );
-    damage -> execute();
-  }
-
+  // Unholy pact ticks twice on buff expiration, hence the following override
```

Comment:
From my limited testing, it seems that unlike Helchains, Unholy Pact seems to tick two additional times at the end of the buff rather than one additional at the start and at the end.